### ISSUE TITLE
action('select')->execute() doesn't use typecastLoadRow normalizer

### ISSUE THREAD

**Issue by DarkSide666** [atk4/data#179]
Currently $persistence->typecastLoadRow only works when you load one particular record.

But in case of grid, for example, we do $model->action('select')->execute(); and in such case $persistance->typecastLoadRow is not executed at all.

Any ideas how to make this typecasting consistent ?

**Comment by romaninsh** [comment]
grid should use either foreach or getIterator

**Comment by DarkSide666** [comment]
Yes, but getIterator doesn't typecast too. Only load(), tryLoad() etc. uses typecastLoad

**Comment by DarkSide666** [comment]
getIterator is fixed here: https://github.com/atk4/data/commit/2ea56ddb00e394a79845b892d4b7025b320152ac

**Comment by mvorisek** [comment]
no longer present
